hc/accounts/tests/test_login.py

```diff
@@ -20,18 +20,21 @@ def test_it_sends_link(self):
         assert r.status_code == 302
 
         ### Assert that a user was created
+        user = User.objects.get(email="alice@example.org")  
+        self.assertEqual(user.email, "alice@example.org")
```

Comment:
good work here, to make the test stronger we should check the count before the post and the count afterwards and include a test that the count of users has increased by one.